### DISCUSSION TITLE
Explanations of Usage Examples in ReadMe?

### DISCUSSION THREAD

**Discussion by SteveMurgaski** [Purfview/whisper-standalone-win#325] [Q&A] [answered]
Hi. I know it's boring to do documentation, but so many people would be interested in this project (like me) but feel overwhelmed by all the command line options. The Github ReadMe gives 4 Usage Examples, but no notes on what the --options are doing.
Could we have more examples, and a sentence or 2 about what each example is doing, maybe a use case it might be recommended for?
Thanks.

**Comment by Purfview** [comment, marked as answer]
> The Github ReadMe gives 4 Usage Examples, but no notes on what the --options are doing.

The 4th usage example doesn't ring any bells?

>...feel overwhelmed by all the command line options

Only 101 at the moment. for example ffmpeg has probably around half thousand.

**Reply by SteveMurgaski** [reply]
Yes, I see that the 4th example prints out the help file.
Even a one-sentence description of what each example does would help people getting started figure out what to try.
"Transcribe speech from a video with a single speaker, minimal background noise, with time stamps in the transcript:
example
Translate speech from multiple speakers to English, ignoring background noise, with time stamps and indications of who is speaking in the transcript:
example..."

Then I could get a sense of what the program can do, because I would try your examples on the appropriate kind of content.

**Reply by Purfview** [reply]
No, those examples are just general show of how command looks so you can quickly grasp the basic general usage.

If you want to dive deeper then read the help, forums, issues, the source codes, academic papers.
